markdownhighlighter.cpp

```diff
@@ -1678,6 +1762,20 @@ void MarkdownHighlighter::highlightAdditionalRules(
                 continue;
             }
 
+            bool contains = false;
+            if (text.contains(rule.shouldContain[0])) {
```

Comment:
I guess all those ifs can be accumulated in one if with some "or" operators :laughing: